Makefile.js

```diff
@@ -648,8 +669,20 @@ target.gensite = function(prereleaseVersion) {
             let text = cat(filename),
                 title;
 
-            // 5. Prepend page title and layout variables at the top of rules
+            // 6. Prepend page title and layout variables at the top of rules
             if (path.dirname(filename).indexOf("rules") >= 0) {
+
+                // Find out if the rule requires a special docs portion (e.g. if it is recommended and/or fixable)
+                const isRecommended = recommended.has(ruleName);
```

Comment:
I don't think we need to use `Set` for both of them? Maybe we should do `rules.get(ruleName).meta.docs.recommended` instead?